rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2869,6 +2869,10 @@ def _check_tokens_existence(user_db_cursor: 'DBCursor', expect_removed: bool = F
                 ('bybit_history_events_cb1', 1641386301, 1641386400),
             ],
         )
+        write_cursor.execute(
```

Comment:
do that directly in the test DB instead of here. And then in the part of the test before the upgrade, check manual is there. The part after the upgrade can stay the same.